pgxn/neon/neon_perf_counters.h

```diff
@@ -15,7 +15,7 @@
 #include "storage/proc.h"
 #endif
 
-static const uint64 getpage_wait_bucket_thresholds[] = {
+static const uint64 io_wait_bucket_thresholds[] = {
```

Comment:
The buckets should probably be different for the LFC operations. We expect LFC read/write to be something like < 100 μs. Waits longer than > 1 s are certainly abnormal and we hardly need so many buckets on that end. On the other hand, more buckets in the 0-20 μs range might provide useful information.